build.gradle

```diff
@@ -2,7 +2,7 @@ buildscript {
     ext {
         kotlinVersion = "1.2.71"
         // MOBILE-3449 Parameters encoding changed in later versions https://github.com/square/okhttp/issues/3235
-        okHttpVersion = "3.9.1"
+        okHttpVersion = "3.11.0"
```

Comment:
I had problems with character encoding in tests when I bumped it - so I guess we can adjust our expected values accordingly.

You can also delete the comment above.